src/Microsoft.Extensions.WebEncoders/Testing/HtmlTestEncoder.cs

```diff
@@ -18,11 +19,31 @@ public override int MaxOutputCharactersPerInputCharacter
 
         public override string Encode(string value)
         {
+            if (value == null)
+            {
+                throw new ArgumentNullException(nameof(value));
+            }
+
+            if (string.IsNullOrEmpty(value))
```

Comment:
`IsNullOrEmpty()` is heavily optimized, `value == string.Empty` often gets the opposite code review comments, and `IsNullOrEmpty()` is what we use everywhere even for known non-`null` values.

So, still prefer `value == string.Empty`?